server/src/app.module.ts

```diff
@@ -1,13 +1,56 @@
-import { Module } from '@nestjs/common';
+import { Module, MiddlewareConsumer, RequestMethod } from '@nestjs/common';
+import { ExpressSessionMiddleware } from '@nest-middlewares/express-session';
+import { CookieParserMiddleware } from '@nest-middlewares/cookie-parser';
 import { ConfigModule } from './config/config.module';
 import { TypeOrmModule } from '@nestjs/typeorm';
+import { TypeormStore } from 'typeorm-store';
+import { getConnection } from 'typeorm';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
 import { CustomerModule } from './modules/Customer/customer.module';
+import { ConfigService } from './config/config.service';
+import { Sessions } from './entity/Sessions';
+import {UserModule} from './modules/User/user.module';
+import {AuthModule} from './modules/Auth/auth.module';
+import {SessionModule} from './modules/Session/session.module';
+import {ClearCookieMiddleware} from './midelwares/clearCookie.middleware';
 
 @Module({
-  imports: [ConfigModule, TypeOrmModule.forRoot(), CustomerModule],
+  imports: [
+    ConfigModule,
+    TypeOrmModule.forRoot(),
+    CustomerModule,
+    UserModule,
+    AuthModule,
+    SessionModule,
+  ],
   controllers: [AppController],
   providers: [AppService],
 })
-export class AppModule {}
+
+export class AppModule {
+  constructor(private readonly configService: ConfigService) {}
+
+  configure(consumer: MiddlewareConsumer) {
+    ExpressSessionMiddleware.configure({
+      secret: this.configService.secretKey(),
+      store: new TypeormStore({
```

Comment:
I would rather store sessions in some "in memory" storages like redis.

Reply:
Last Saturday, we decided using TypeORM as the main approach for everything. Something was changed?